test/source/util/index.ts

```diff
@@ -51,35 +50,41 @@ export type TestKeyInfoWithFilepath = TestKeyInfo & { filePath?: string, expired
 
 interface TestSecretsInterface {
   ci_admin_token: string;
-  auth: { google: { email: string, password?: string, secret_2fa?: string }[], };
-  keys: TestKeyInfo[];
+  auth: { google: { email: string, password?: string, secret_2fa?: string }[] };
 }
 
 export class Config {
 
   public static extensionId = '';
 
   private static _secrets: TestSecretsInterface;
+  private static _keys: TestKeyInfo[];
 
   public static secrets = (): TestSecretsInterface => {
     if (!Config._secrets) {
       try {
         Config._secrets = JSON.parse(fs.readFileSync('test/test-secrets.json', 'utf8'));
-        Config._secrets.keys = testKeyConstants.keys;
       } catch (e) {
         console.error(`skipping loading test secrets because ${e}`);
-        Config._secrets = { auth: { google: [] }, keys: [] } as any as TestSecretsInterface;
+        Config._secrets = { ci_admin_token: "", auth: { google: [] } };
```

Comment:
@tomholub I think we should define eslint rule which discourages any. I think any type is not good in any case.

`no-explicit-any`